src/diffusers/pipelines/hunyuandit/pipeline_hunyuandit.py

```diff
@@ -239,6 +240,8 @@ def __init__(
             else 128
         )
 
+        self.unet_name = 'transformer' # to support load_lora_weights
```

Comment:
Hmm, this feels a little weird to me. But okay for now. I will attempt to refactor this to harmonize. Cc: @yiyixuxu